lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -292,11 +292,17 @@ def max_thread_limit
     end
     next if File::exists?(target)
 
+    # https://tickets.puppetlabs.com/browse/PUP-4884
+    working_dir = File.expand_path('spec/fixtures/work-dir')
+    working_dir = working_dir.gsub(/\\/, '/')
+    target_dir = File.expand_path('spec/fixtures/modules')
+    target_dir = target_dir.gsub(/\\/, '/')
```

Comment:
@DavidS let's chat here. I mentioned that the gsub bits are maybe a bit of overkill and probably unnecessary. This is fixing the issue with PUP-4884 in that it doesn't switch the paths to forward slashes by switching the paths to forward slashes here. I don't have a mistrust for File.expand_path, but maybe you didn't quite catch my humor here.